packages/frontend/components/Provenance/Feed.vue

```diff
@@ -39,9 +39,7 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>. -->
             
             <div v-for="(attachment, i) in attachmentURLs[index.toString()]" :key="i">
                 <!-- Image -->
-                 <div v-if="((attachment.fileName).toLowerCase()).endsWith('.jpg') || ((attachment.fileName).toLowerCase()).endsWith('.png')  ">
-                    <img :src="attachment.url" :alt="Image" style="width: 150px; padding: 5px;" data-bs-toggle="modal" data-bs-target="#imageModal" @click="modalImage = attachment.url">
```

Comment:
Can you please include the full img tag because the end part opens a modal with a larger image. This part should be added back: data-bs-toggle="modal" data-bs-target="#imageModal" @click="modalImage = attachment.url"